### DISCUSSION TITLE
Are there any tablets that !lazer is specifically being tested with, or optimized for?

### DISCUSSION THREAD

**Discussion by Silquetoast** [ppy/osu#20412] [Q&A] [answered]
I'm new here so I don't know if this is the place to ask this, but I felt like you guys at least should be pretty knowledgeable on the subject.

My tablet is getting old and worn down, so before I jump straight to buying a new one I wanted to ask... Are there were any tablets that !lazer might be consistently tested with or optimized for? 

Thanks in advance

**Comment by NotGumballer91** [comment, marked as answer]
IIRC, Lazer uses OpenTabletDriver. You can see the list of [supported tablets](https://opentabletdriver.net/Tablets) here.
